.github/workflows/OCV-PR-4.x-iOS.yaml

```diff
@@ -0,0 +1,54 @@
+name: OCV PR:4.x iOS
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/OCV-PR-4.x-iOS.yaml'
+  workflow_call:
+  schedule:
+  - cron: "0 0 * * *"
+
+env:
+  PR_AUTHOR: ${{ github.event.pull_request.user.login }}
+  PR_AUTHOR_FORK: ${{ github.event.pull_request.head.repo.full_name }}
+  SOURCE_BRANCH_NAME: ${{ github.head_ref }}
+  TARGET_BRANCH_NAME: ${{ github.base_ref }}
+  GIT_CACHE: /Users/xperience/git_cache
+
+jobs:
+  Build:
+    runs-on: macOS-M1
```

Comment:
Temporary this device. The goal to use devices by OpenCV-CN, it will be faster. However, there are issues during the build now.